resources/qml/Menus/ConfigurationMenu/CustomConfiguration.qml

```diff
@@ -64,17 +64,18 @@ Item
             anchors.left: parent.left
         }
 
-        OldControls.ToolButton
+        Button
         {
             id: printerTypeSelector
             text: Cura.MachineManager.activeMachine !== null ? Cura.MachineManager.activeMachine.definition.name: ""
-            tooltip: text
+
             height: UM.Theme.getSize("print_setup_big_item").height
             width: Math.round(parent.width * 0.7) + UM.Theme.getSize("default_margin").width
             anchors.right: parent.right
-            style: UM.Theme.styles.print_setup_header_button
+            onClicked: menu.open()
+            //style: UM.Theme.styles.print_setup_header_button
 
-            menu: Cura.PrinterTypeMenu { }
+            Cura.PrinterTypeMenu { id: menu}
```

Comment:
Isn't there logic removed here? such as the tooltip text. and the theme styling?

Reply:
Styling doesn't exist in controls 2 anymore.  

The tooltip was just showing the same as the text on the button, so i figured i could safely remove this. The only reason that I touched this button here is because it had a menu (The new button makes it a bit harder to add a tooltip, so I didn't feel that it was really worth it. But I will re-add it if you feel that it still is important